tools/OpenAPIParser/OpenAPIParser.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Why not use hidi with the validate command instead?

Reply:
This (what kiota and DevX API do) was the quickest I could find at 7pm while dealing with the ICM and the time pressure to block today's generation with a gate :) I opened an issue to address this referencing this comment.

Reply:
thanks for dealing with this! I do understand the urgency aspect of this PR.